actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -59,7 +59,7 @@ module TranslationHelper
       # they can provide HTML values for.
       def translate(key, options = {})
         options = options.dup
-        if options.has_key?(:default)
+        if options.has_key?(:default) && !options[:default].nil?
```

Comment:
Do you know what happen if you do `default: false`? Does this get properly handled by I18n but also breaks in Rails?

If so, it might be worth fixing that and adding a failing test case to prevent further regression.